lib/add-tests.coffee

```diff
@@ -1,5 +1,5 @@
 async = require 'async'
-_ = require 'underscore'
+_ = require 'lodash'
```

Comment:
It's fine for me to replace underscore with lodash. We should make it in every file including package.json. This changes help be consistent.